File: SuboptimalProcedureLearner/SuboptimalProcedureLearner/Misc/ListTools.cs
using System;
using System.Collections.Generic;


namespace Misc
{
    class ListTools
    {
        public static bool isListTheSameInt(List<int> a, List<int> b)
        {
            int i;
            
            if( a.Count != b.Count )
            {
                return false;
            }

            for( i = 0; i < a.Count; i++ )
            {
                if( a[i] != b[i] )
                {
                    return false;
                }
            }

            return true;
        }
    }
}

File: src/java/ptrman/agix0/src/java/ptrman/misc/ListTools.java
package ptrman.misc;

import java.util.List;

public class ListTools {
    public static boolean isListTheSameInt(List<Integer> a, List<Integer> b) {
        int i;
        if( a.size() != b.size() ) {
            return false;
        }
         
        for (i = 0;i < a.size();i++) {
            if( a.get(i) != b.get(i) ) {
                return false;
            }
        }
        return true;
    }
}

